src/azure-cli/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -618,3 +637,50 @@ def cli_returnnsdetails(cmd, resource_group_name, namespace_name, max_size_in_me
                                                                             40960, 81920]:
         raise CLIError(
             '--max-size on Premium sku namespace only supports upto [1024, 2048, 3072, 4096, 5120, 10240, 20480, 40960, 81920] GB')
+
+
+# Private Endpoint
+def _update_private_endpoint_connection_status(cmd, client, resource_group_name, namespace_name,
+                                               private_endpoint_connection_name, is_approved=True, description=None):
+    from azure.core.exceptions import HttpResponseError
+    import time
+
+    PrivateEndpointServiceConnectionStatus = cmd.get_models('PrivateLinkConnectionStatus')
+
+    private_endpoint_connection = client.get(resource_group_name=resource_group_name, namespace_name=namespace_name,
+                                             private_endpoint_connection_name=private_endpoint_connection_name)
+
+    old_status = private_endpoint_connection.private_link_service_connection_state.status
+    if old_status != "Approved" or not is_approved:
+        private_endpoint_connection.private_link_service_connection_state.status = PrivateEndpointServiceConnectionStatus.APPROVED\
+            if is_approved else PrivateEndpointServiceConnectionStatus.REJECTED
+        private_endpoint_connection.private_link_service_connection_state.description = description
```

Comment:
We should be updating description regardless of what updated status is right ? Shouldn't this be outside if block. and we should be updating it only if the user has given description and not other times.